src/wp-admin/includes/file.php

```diff
@@ -1962,9 +1962,10 @@ function move_dir( $from, $to ) {
 
 	$result = false;
 
-	if ( 'direct' === $wp_filesystem->method && ! $wp_filesystem->is_virtual_box() ) {
+	if ( ! $wp_filesystem->is_virtual_box() ) {
 		$wp_filesystem->rmdir( $to );
-		$result = @rename( $from, $to );
+
+		$result = $wp_filesystem->move( $from, $to );
```

Comment:
The whole problem with `$wp_filesystem->move()` is that it's not a recursive copy this will never follow on to `copy_dir()` fallback.